File: TCC-Blockchain/TCC-Blockchain/Dtos/ResponseLicitacaoDto.cs
namespace TCC_Blockchain.Dtos; 

public class ResponseLicitacaoDto
{
    public string Titulo { get; set; }
    public string Descricao { get; set; }
    public string HashETP { get; set; }
    public string HashEdital { get; set; }
    public ulong DataInicio { get; set; }
    public byte Estagio { get; set; }

}

File: Backend/app/Interfaces/response_licitacao_dto.ts
export default class ResponseLicitacaoDto {
    titulo: string = '';;
    descricao: string = '';;
    hashETP: string = '';;
    hashEdital: string = '';;
    dataInicio: Date = new Date();
    dataInicioCandidaturas: Date = new Date();
    dataFimCandidaturas: Date = new Date();
    estagio: string = '';
    enderecoBloco: string = '';
}